tests/integration/generate/generation_framework.py

```diff
@@ -32,7 +32,7 @@ def generate_and_check():
 
         assert os.path.getsize(tmp_file) > 1000, 'File {} should not be empty.'.format(tmp_file)
 
-        if fmt.lower() == 'pdf':
+        if fmt.lower() == 'pdf' and sys.platform != 'darwin':
```

Comment:
If I understand this correctly, this change will make tests blindly pass on MacOS?

I think it would be better to leave the tests failing and properly fix them in another PR.

Reply:
The issue is that pdfinfo is not available on Mac OS X. This is an additional check that the pdf file is meaningful, on top of checking the file is not empty. Included warning message, but keeping the different treatment on mac.